Bylaws.md

```diff
@@ -104,3 +104,42 @@ The purpose of _The Polytechnic_ shall be to keep the members of the Rensselaer
     1. In the event of a conflict of interest or the appearance of a conflict of interest, an individual must recuse themselves, thereby not counting towards voting quorum. Abstentions, which count towards quorum, may be used if an individual does not feel that they have enough information. An abstention has the same effect as a no vote.
     
     1. Absentee ballots and voting by proxy are not allowed for editorial content meetings.
+1. Endorsement Meetings
+    
+    1. Endorsement meetings shall be held for the purpose of deciding whether to endorse a candidate for a campus officer position.
+        1.	The editorial board may consider endorsing candidates for any number of positions, but in order to consider any candidates for endorsement of any position, the editorial board must at least consider the endorsement of a candidate for Grand Marshal and for President of the Union.
+        
+        1.	The editorial board shall provide the option of an interview to each candidate as recognized by the Student Senate running for an office up for endorsement and present to them prior to the interview a written questionnaire. The editorial board must consider each of those candidates who are interviewed for endorsement.
+        
+        1.	Meetings during which candidates are interviewed shall be closed to all except those outlined in Section (IV) subsection (3) clause (iii) and the interviewee. Quorum shall consist of two-thirds of the editorial board membership. 
```

Comment:
Why not just add here instead of the confidentiality thing "The editorial board may publish transcripts or recordings of these interviews as part of its coverage of elections."